projects/core/src/product/facade/product.service.ts

```diff
@@ -20,7 +21,8 @@ import { ProductSelectors } from '../store/selectors/index';
 export class ProductService {
   constructor(
     protected store: Store<StateWithProduct>,
-    protected productLoading: ProductLoadingService
+    protected productLoading: ProductLoadingService,
+    private productConnector: ProductConnector
```

Comment:
MAJOR: please use inject() & not constructor injection as suggested in coding guideline. recheck all other files for the same..

Reply:
done